master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
But I read the documentation and c['services'] is for allowing advanced users to add custom services to Buildbot at runtime - not for a permanent service like the stats service, I believe.